apps/admin/frontend/src/screens/contest_adjudication_screen.tsx

```diff
@@ -317,13 +329,29 @@ export function ContestAdjudicationScreen(): JSX.Element {
 
   const adjudicateCvrContestMutation = adjudicateCvrContest.useMutation();
 
-  const officialOptions = useMemo(
-    () =>
-      isCandidateContest
+  const officialOptions = useMemo(() => {
+    if (!cvrVoteInfoQuery.data) {
+      // Return a placeholder that will be replaced once data loads
```

Comment:
Could we remove the useMemo and move this whole thing down to after the guard that checks that all the data has loaded? Then we wouldn't need a placeholder. I'm not sure how important the memoization is for performance, but it doesn't seem like this would be a very expensive calculation since there are only a small number of candidates per contest

Reply:
we need this argument for the useContestAdjudicationState hook so I think we need a placeholder regardless

Reply:
Oh gotcha. Well I'd probably pull the data loading out into a parent component in that case but that refactor might be more scope than makes sense for this PR